src/legacy/index.ts

```diff
@@ -314,8 +323,10 @@ async function buildSubtree(
         addLabel(depTree, 'pruned', 'true');
       }
       return depTree;
+    }else {
```

Comment:
missing a white space. did you `npm run format` ?

Reply:
:O 
fixed!